packages/components/src/templates/badge.stories.ts

```diff
@@ -80,9 +98,12 @@ export const NavigationItemWithBadge = {
             <img class="h-8" src="images/logo-unioninvestment-lg.svg" alt="Logo" />
           </a>
           <div class="flex">
-            <sd-navigation-item class="relative">
+            <sd-navigation-item class="relative" role="button">
```

Comment:
That's weird. The element should be a button already. Is this still needed?

Reply:
Removed